lib/liquid/extensions.rb

```diff
@@ -25,6 +26,12 @@ def to_liquid
   end
 end
 
+class BigDecimal
+  def to_liquid
+    to_f
```

Comment:
I mean the way `{{ decimal }}` and `{{ decimal | abs }}` are rendered isn't consistent.  I suppose it would make the abs filter consistent with other math filters.